feature/experimental/system/gnmi/benchmarking/ate_tests/drained_configuration_convergence_time/drained_configuration_convergence_time_bgp_test.go

```diff
@@ -55,6 +56,11 @@ func setMED(t *testing.T, dut *ondatra.DUTDevice, d *oc.Root) {
 	// setMedBGP := actions5.GetOrCreateBgpActions().GetOrCreateSetMed()
 	// setMedBGP.SetMed = ygot.Uint32(bgpMED)
 	actions5.GetOrCreateBgpActions().SetLocalPref = ygot.Uint32(100)
+
```

Comment:
I ran into the following issue: 

drained_configuration_convergence_time_bgp_test.go:172: GetAll(t) on ate([ixia03-den86.corp.google.com](http://ixia03-den86.corp.google.com/)) at /network-instances/network-instance[name=2/7]/protocols/protocol[identifier=BGP][name=0]/bgp/rib/afi-safis/afi-safi[afi-safi-name=IPV4_UNICAST]/ipv4-unicast/neighbors/neighbor[neighbor-address=*]/adj-rib-in-pre/routes/route[path-id=0][prefix=*]/state/prefix: query "/network-instances/network-instance[name=2/7]/protocols/protocol[identifier=BGP][name=0]/bgp/rib/afi-safis/afi-safi[afi-safi-name=IPV4_UNICAST]/ipv4-unicast/neighbors/neighbor[neighbor-address=*]/adj-rib-in-pre/routes/route[path-id=0][prefix=*]/state/prefix": value not present

We need to revise the code at line 172-173; https://github.com/openconfig/featureprofiles/pull/1189/files is an example.
			pref := gnmi.GetAll(t, ate, prefixPath.State())
			asPath := gnmi.GetAll(t, ate, rib.AttrSetAny().AsSegmentAny().State())

Reply:
@guoshiuan 
Changed to WatchAll instead of GetAll